sphinx/source/index.rst

```diff
@@ -21,6 +21,7 @@ Captum API Reference
    influence
    utilities
    base_classes
+   approximation_methods
```

Comment:
I don't think we want to expose this. People are not allowed to access it through public api

Reply:
Okay, I've removed it